cli/medperf/comms/rest.py

```diff
@@ -329,18 +328,18 @@ def get_cube_metadata(self, cube_uid: int) -> dict:
             raise CommunicationRetrievalError("the specified cube doesn't exist")
         return res.json()
 
-    def get_cube(self, url: str, cube_uid: int) -> str:
+    def get_cube(self, url: str, cube_path: str) -> str:
```

Comment:
Was there any specific reason for this change? I like it! It means the comms layer no longer cares how to build the cube path. Did you encounter any issues that got fixed by this? Or was it intuition?

Reply:
These changes should be reflected in the comms interface as well. Just copy-paste the signature and docstring

Reply:
Yes! The reason was this:
In the cube entity, we have download methods. These methods were calling the corresponding comms methods by providing the cube uid. However, after our recent modification of introducing cube uid and cube generated uid, we are now having two possibilities for the path where cube assets are downloaded (before, or,  after submission). We should then in each Cube.download* method check if the cube is submitted then pass the server uid to the comms method, else pass the generated uid.
It seems ugly :) + as you said, we got the benefit of making sure comms should not contain logic of constructing cube path, which is ideal.

Reply:
This was causing a failure, I forgot where. The cube assets where downloaded in `cubes/<server_uid>` which was actually `cubes/None`